packages/utils/telemetry-utils/src/logger.ts

```diff
@@ -147,7 +151,7 @@ export abstract class TelemetryLogger implements ITelemetryLogger {
             category: event.category ?? (error === undefined ?  "generic" : "error"),
         };
         if (error !== undefined) {
-            TelemetryLogger.prepareErrorObject(newEvent, error, false);
```

Comment:
This used to be true sometimes but was lost in a prior refactor.  No reason not to always generate if missing.